front/app/containers/Admin/projects/project/analysis/Insights/Summary.tsx

```diff
@@ -69,6 +76,24 @@ const Summary = ({ summary }: Props) => {
   };
 
   const hasFilters = !!Object.keys(summary.attributes.filters).length;
+  const tagId = summary.attributes.filters.tag_ids
+    ? summary.attributes.filters.tag_ids[0]
+    : null;
+
+  const tag = tags?.data.find((tag) => tag.id === tagId);
```

Comment:
Maybe we could already do `tags`? Probably less code as its aligned with how the filters are defined, and ready for multiselect.